Applications/AlexNet/jni/main.cpp

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file   main.cpp
+ * @date   25 Jul 2022
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com> / Donghak Park
```

Comment:
Might be better to use new line here. 
@author Jijoong Moon ...
@author Donghak Park ..

Reply:
FIX